cloud/amazon/ec2_vpc.py

```diff
@@ -656,6 +656,28 @@ def terminate_vpc(module, vpc_conn, vpc_id=None, cidr=None):
 
     return (changed, vpc_dict, terminated_vpc_id)
 
+def get_vpc (module, vpc_conn, vpc_id):
```

Comment:
I think you should pass in cidr in case people want to use the cidr + tags feature of `find_vpc`.